### PR TITLE
FLIP for Errant ANs

### PR DESCRIPTION
**Pull request by KshitijChaudharyHBS**

### CODE REVIEW

File: governance/20230105-identify-errant-node.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Identify Errant Access Nodes
+
+| Status        | WIP                                            |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [NNN](https://github.com/onflow/flow/pull/NNN) (update when you have PR #)|
+| **Author(s)** | James Lofgren (james.lofgren@coinbase.com)           |
```
Comment (TheUnseen12) [inline governance/20230105-identify-errant-node.md:6]: Can we update the email to use '76029851+TheUnseen12@users.noreply.github.com'

Comment (vishalchangrani) [inline governance/20230105-identify-errant-node.md:6]:
```suggestion
| **Author(s)** | James Lofgren (76029851+TheUnseen12@users.noreply.github.com)           | 
```

File: governance/20230105-identify-errant-node.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Identify Errant Access Nodes
+
+| Status        | WIP                                            |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [NNN](https://github.com/onflow/flow/pull/NNN) (update when you have PR #)|
```
Comment (vishalchangrani) [inline governance/20230105-identify-errant-node.md:5]:
```suggestion
| **FLIP #**    | [64](https://github.com/onflow/flow/pull/64) (update when you have PR #)|
```

File: governance/20230105-identify-errant-node.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# Identify Errant Access Nodes
+
+| Status        | WIP                                            |
```
Comment (vishalchangrani) [inline governance/20230105-identify-errant-node.md:3]:
```suggestion
| Status        | Drafted                                            |
```

File: governance/20230105-identify-errant-node.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+# Identify Errant Access Nodes
+
+| Status        | Drafted                                            |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [64](https://github.com/onflow/flow/pull/64) (update when you have PR #)|
```
Comment (KshitijChaudharyHBS) [inline governance/20230105-identify-errant-node.md:5]:
```suggestion
| **FLIP #**    | [64](https://github.com/onflow/flow/pull/64)
```

File: governance/20230105-identify-errant-node.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
+# Identify Errant Access Nodes
```
Comment (KshitijChaudharyHBS) [inline governance/20230105-identify-errant-node.md:1]:
```suggestion
# Identifying and Ejecting Errant Access Nodes
```

File: governance/20230105-identify-errant-node.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+At the end of the registration cycle, once interested operators have acquired the slots and submitted (*at least*) the minimum required stake, some operators are randomly selected from the list, allowing them to run an AN in the subsequent epoch. This process can be enhanced in the future with an auction process for selecting node operators in a fully permissionless manner. 
+
+The number of nodes selected in each epoch will be in conjunction with the maximum number of permissionless ANs that are allowed to operate in the network at a time, which is initially fixed at 5. This is in addition to the 122 permissioned access nodes already in the system.
+
+**Access Node Configurations**
+
+- Minimum stake: 100 Flow
```
Comment (KshitijChaudharyHBS) [inline governance/20230105-identify-errant-node.md:26]:
```suggestion
- Minimum stake: 100 FLOW
```

File: governance/20230105-identify-errant-node.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
+# Identifying and Ejecting Errant Access Nodes
+
+| Status        | Drafted                                            |
```
Comment (KshitijChaudharyHBS) [inline governance/20230105-identify-errant-node.md:3]:
```suggestion
| Status        | Proposed                                            |
```

**Review by vishalchangrani — APPROVED** [review]

**Pull request merged**